components/inputs/test/input-search.test.js

```diff
@@ -103,7 +103,7 @@ describe('d2l-input-search', () => {
 
 			let clock;
 			beforeEach(() => {
-				clock = useFakeTimers();
+				clock = useFakeTimers({ toFake: ['clearTimeout', 'setTimeout'] });
```

Comment:
The new `fixture` relies on `await nextFrame()` when it resets stuff, so we need to be more targeted with our mocking otherwise `requestAnimationFrame` gets mocked.

Reply:
Yep I've already included it in the [draft upgrade guide](https://desire2learn.atlassian.net/wiki/spaces/GAUD/pages/3667394686/Upgrade+Guide+draft), but I could definitely see it going into a README as well.